ci-operator/config/monstorak/monstorak-operator/monstorak-monstorak-operator-master.yaml

```diff
@@ -0,0 +1,73 @@
+canonical_go_repository: github.com/monstorak/monstorak-operator
+
+base_images:
+  os:
+    cluster: https://api.ci.openshift.org
+    name: centos
+    namespace: openshift
+    tag: '7'
+  operator-registry:
+    name: "4.0"
+    namespace: ocp
+    tag: operator-registry
+
+resources:
+  '*':
+    limits:
+      memory: 2Gi
+    requests:
+      memory: 250Mi
+      cpu: 100m
+
+images:
+- from: os
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/monstorak/monstorak-operator/out/operator 
+  dockerfile_path: openshift-ci/Dockerfile.deploy
+  to: monstorak
+- from: operator-registry
+  dockerfile_path: openshift-ci/Dockerfile.registry.intermediate
+  to: operator-registry-base
+- from: operator-registry-base
+  dockerfile_path: openshift-ci/Dockerfile.registry.build
+  to: monstorak-operator-registry
+# - from: src
```

Comment:
Please avoid leaving comments in this file. If they are needed, please leave a comment with the reason also.